File: JAVA/Employee salary/com/Salary/Person.java
package com.Salary;


public abstract class Person {
    private String name;
    private String lastname;
    private int age;
    private int identification;
    

    public Person(String name, String lastname, int age, int identification) {
        this.name = name;
        this.lastname= lastname;
        this.age = age;
        this.identification = identification;   
        
    }

    public String getName() {
        return this.name;
    }

    public String getLastname() {
        return this.lastname;
    }

    public int getAge() {
        return this.age;
    }

    public int getIdentification() {
        return this.identification;
    }

    public abstract String getDescription();
    public abstract double getNetSalary();
}

File: PHP/Employee Salary/Salary/Person.php
<?php


abstract class Person {
    private string $name;
    private string $lastname;
    private int $age;
    private int $identification;

    public function __construct(string $name, string $lastname, int $age, int $identification) {
        $this->name = $name;
        $this->lastname = $lastname;
        $this->age = $age;
        $this->identification = $identification;
    }

    public function getName(): string {
        return $this->name;
    }

    public function getLastname(): string {
        return $this->lastname;
    }

    public function getAge(): int {
        return $this->age;
    }

    public function getIdentification(): int {
        return $this->identification;
    }

   
    abstract public function getDescription(): string;
    abstract public function getNetSalary(): float;
}